web/src/features/hfiCalculator/components/meanIntensity.ts

```diff
@@ -20,10 +20,12 @@ export const calculateMeanIntensityGroup = (
   }
   return stationIntensityGroups.length === 0
     ? undefined
-    : Math.round(
-        (10 * stationIntensityGroups.reduce((a, b) => a + b, 0)) /
-          stationIntensityGroups.length
-      ) / 10
+    : Math.ceil(
+        Math.round(
```

Comment:
I don't think it's necessary to do both Math.round() and Math.ceil() - shouldn't Math.ceil() alone produce the desired output?

Reply:
`Math.round` is around the intermediate operation, it's then divided by 10. `Math.ceil` is around the last operation. It would change the behaviour but seems reasonable.

Reply:
Removed `round`.